source/get-started/bootable-usb.rst

```diff
@@ -39,64 +39,53 @@ Burn the |CL| image onto a USB drive
-      sdc      8:32   1   7.3G  0 disk
-      └─sdc1   8:33   1   7.3G  0 part /run/media/user1/LINUX MINT
-      sda      8:0    0 335.4G  0 disk
-      ├─sda4   8:4    0    28G  0 part
-      ├─sda2   8:2    0   3.7G  0 part [SWAP]
-      ├─sda7   8:7    0     6G  0 part /home
-      ├─sda5   8:5    0     1G  0 part /boot
-      ├─sda3   8:3    0   954M  0 part /boot/efi
-      ├─sda1   8:1    0    28G  0 part
-      ├─sda8   8:8    0    30G  0 part /
-      └─sda6   8:6    0   7.9G  0 part [SWAP]
+
+      NAME          SIZE VENDOR   MODEL                    TRAN   TYPE PARTLABEL                    MOUNTPOINT
+      /dev/sda    119.2G ATA      SAMSUNG_MZ7PC128HAFU-000 sata   disk                              
+      ├─/dev/sda1   450M                                          part Basic data partition         
+      ├─/dev/sda2   100M                                          part EFI system partition         
+      ├─/dev/sda3    16M                                          part Microsoft reserved partition 
+      ├─/dev/sda4  97.2G                                          part Basic data partition         
+      ├─/dev/sda5   142M                                          part EFI                          
+      ├─/dev/sda6   245M                                          part linux-swap                   [SWAP]
+      └─/dev/sda7  21.1G                                          part /                            /
+      /dev/sdb      7.5G General  UDisk                    usb    disk                              
+      └─/dev/sdb1   7.5G                                          part Microsoft Basic Data         /run/media/clear/CENA_X64FRE
+      /dev/sdc       15G          Patriot_Memory           usb    disk                              
+      └─/dev/sdc1    15G                                          part                              /run/media/clear/U
 
    .. note::
 
       Some Linux distros may automatically mount a USB drive when it is plugged in.
 
 #. If the USB drive you want to use is mounted, it must be umounted before
    burning an image onto it.  Use the :command:`umount` command followed by
-   the device identifier/partition. For example, to unmount all /dev/sdd
-   partitions:
+   the device identifier/partition. For example, to unmount all of the 
+   ``/dev/sdc`` partitions:
 
    .. code-block:: bash
 
-      sudo umount /dev/sdd*
+      sudo umount /dev/sdc*
 
-#. Burn the image onto the USB drive. This example burns an image onto `/dev/sdd`.  The device name of the USB may vary.
+#. Burn the image onto the USB drive. This example burns an image onto ``/dev/sdc``.  
+   The device name of the USB may vary.
 
    .. code-block:: bash
 
-      sudo dd if=./clear-[version number]-live-[desktop | server].iso of=/dev/sdd oflag=sync bs=4M status=progress
+      sudo dd if=./clear-[version number]-live-[desktop | server].iso of=/dev/sdc oflag=sync bs=4M status=progress
 
```

Comment:
See Ln 182 (revised) below.  Currently, the Rufus instruction is missing one step. 
Replace 182 with: 
#. Under boot selection , click the :guilabel:`SELECT` button.
#. In the dialog, navigate to where the |CL| iso image was downloadedand select it.